shared/types/field/tableField.ts

```diff
@@ -0,0 +1,20 @@
+import { FieldBase, BasicField } from './base'
+import { DropdownFieldBase } from './dropdownField'
+import { ShortTextFieldBase } from './shortTextField'
+
+export interface ShortTextColumnBase extends ShortTextFieldBase {
```

Comment:
I'm not sure this is right, we don't have `fieldType` for table columns right?

Reply:
Good catch, I only saw the discriminator function, grrr and forgot that the fieldType is discriminated against and not in the value itself. Will fix

Reply:
Fixed in 395149a

Reply:
Fixed in ca42de9